### PR TITLE
adding aurora and rds changes to move to tf12

### PR DESCRIPTION
**Pull request by venturel**
This will be a major version bump

### CODE REVIEW

**Review by minniux — APPROVED** [review]

File: aurora/main.tf [hunk 1]
```diff
@@ -1,38 +1,42 @@
+terraform {
```
Comment (iuriaranda) [inline aurora/main.tf]: Can you move this to a `versions.tf`  file? It's how the migration tool sets it and we're keeping it that way in other modules we've migrated

File: aurora/main.tf [hunk 2]
```diff
@@ -44,26 +48,26 @@ resource "aws_db_parameter_group" "aurora_mysql" {
...
-  availability_zones              = ["${data.aws_availability_zones.available.names}"]
-  db_subnet_group_name            = "${aws_db_subnet_group.aurora.id}"
-  vpc_security_group_ids          = ["${aws_security_group.sg_aurora.id}"]
-  storage_encrypted               = "${var.storage_encrypted}"
-  apply_immediately               = "${var.apply_immediately}"
-  db_cluster_parameter_group_name = "${var.cluster_parameter_group_name}"
-  engine                          = "${var.engine}"
-  engine_version                  = "${var.engine_version}"
-  enabled_cloudwatch_logs_exports = ["${var.enabled_cloudwatch_logs_exports}"]
+  availability_zones              = [data.aws_availability_zones.available.names]
```
Comment (iuriaranda) [inline aurora/main.tf]: I think you should remove the list notation here, the `data.aws_availability_zones.available.names` is already a list afaik

File: aurora/main.tf [hunk 3]
```diff
@@ -44,26 +44,26 @@ resource "aws_db_parameter_group" "aurora_mysql" {
 }
 
 resource "aws_rds_cluster" "aurora" {
-  cluster_identifier              = "${var.project}-${var.environment}${var.tag}-aurora"
-  master_username                 = "${var.rds_username}"
-  master_password                 = "${var.password}"
-  backup_retention_period         = "${var.backup_retention_period}"
-  skip_final_snapshot             = "${var.skip_final_snapshot}"
+  cluster_identifier              = "${var.project}-${var.environment}${var.tag}-aurora}"
```
Comment (iuriaranda) [inline aurora/main.tf]: There's an extra `}` at the end

File: snapshot-replicator/main.tf [hunk 4]
```diff
@@ -47,19 +48,20 @@ resource "aws_iam_policy" "rds_lambda_copy" {
...
 }
 EOF
+
 }
 
 resource "aws_iam_role_policy_attachment" "attach_lambda_copy_policy_to_role" {
-  count      = "${var.enable ? 1 : 0}"
-  role       = "${aws_iam_role.iam_for_lambda.name}"
-  policy_arn = "${aws_iam_policy.rds_lambda_copy.arn}"
+count      = var.enable ? 1 : 0
```
Comment (iuriaranda) [inline snapshot-replicator/main.tf]: indentation here is a bit messed up

**Review by iuriaranda — APPROVED** [review]
A part from the small comments I posted, it looks ok to me

**Pull request merged**